.dockerignore

```diff
@@ -0,0 +1,7 @@
+.devcontainer
+.github
+.mypy_cache
```

Comment:
why is mypy_cache in here?

Reply:
This file became obsolete because we don't build our own dockerfile.